zerver/lib/narrow.py

```diff
@@ -408,47 +425,49 @@ def _pg_re_escape(self, pattern: str) -> str:
                     s[i] = "\\" + c
         return "".join(s)
 
-    def by_stream(
+    def by_channel(
         self, query: Select, operand: Union[str, int], maybe_negate: ConditionTransform
     ) -> Select:
         self.check_not_both_channel_and_dm_narrow(is_channel_narrow=True)
 
         try:
             # Because you can see your own message history for
-            # private streams you are no longer subscribed to, we
+            # private channels you are no longer subscribed to, we
             # need get_stream_by_narrow_operand_access_unchecked here.
-            stream = get_stream_by_narrow_operand_access_unchecked(operand, self.realm)
+            channel = get_stream_by_narrow_operand_access_unchecked(operand, self.realm)
```

Comment:
This local variable rename may be a bit risky if we're not also renaming `class Stream`.

Reply:
I did these backend commits early on in the rename process and was in the focused on deprecating the stream and streams operators. So, the only remaining cases of `stream` in this file are either functions from  other files, references to the legacy operators or the message type.

<details>
<summary>git grep of "stream" in narrow.py after these changes</summary>

```console
$ git grep stream zerver/lib/narrow.py
zerver/lib/narrow.py:from zerver.lib.streams import (
zerver/lib/narrow.py:    can_access_stream_history_by_id,
zerver/lib/narrow.py:    can_access_stream_history_by_name,
zerver/lib/narrow.py:    get_public_streams_queryset,
zerver/lib/narrow.py:    get_stream_by_narrow_operand_access_unchecked,
zerver/lib/narrow.py:    get_web_public_streams_queryset,
zerver/lib/narrow.py:from zerver.models.streams import get_active_streams
zerver/lib/narrow.py:# "stream" is a legacy alias for "channel"
zerver/lib/narrow.py:channel_operators: List[str] = ["channel", "stream"]
zerver/lib/narrow.py:# "streams" is a legacy alias for "channels"
zerver/lib/narrow.py:channels_operators: List[str] = ["channels", "streams"]
zerver/lib/narrow.py:        # or term == {'operator': 'streams', 'operand': 'web-public', 'negated': False}
zerver/lib/narrow.py:                if message["type"] != "stream":
zerver/lib/narrow.py:                if message["type"] != "stream":
zerver/lib/narrow.py:                if message["type"] != "stream":
zerver/lib/narrow.py:            # "stream" is a legacy alias for "channel"
zerver/lib/narrow.py:            "stream": self.by_channel,
zerver/lib/narrow.py:            # "streams" is a legacy alias for "channels"
zerver/lib/narrow.py:            "streams": self.by_channels,
zerver/lib/narrow.py:                | UserMessage.flags.stream_wildcard_mentioned.mask
zerver/lib/narrow.py:            # need get_stream_by_narrow_operand_access_unchecked here.
zerver/lib/narrow.py:            channel = get_stream_by_narrow_operand_access_unchecked(operand, self.realm)
zerver/lib/narrow.py:            matching_channels = get_active_streams(self.realm).filter(
zerver/lib/narrow.py:            recipient_queryset = get_public_streams_queryset(self.realm)
zerver/lib/narrow.py:            recipient_queryset = get_web_public_streams_queryset(self.realm)
zerver/lib/narrow.py:                    include_history = can_access_stream_history_by_name(user_profile, operand)
zerver/lib/narrow.py:                    include_history = can_access_stream_history_by_id(user_profile, operand)
zerver/lib/narrow.py:                and user_profile.can_access_public_streams()
zerver/lib/narrow.py:                return get_stream_by_narrow_operand_access_unchecked(term["operand"], realm)
```
</details>

Reply:
Made no changes here in the most recent updates.

Reply:
Yeah this seems fine to think of as a "file-level" transition.